### ISSUE TITLE
version 4

### ISSUE THREAD

**Issue by anuragmishra81** [marvinroger/homie-esp8266#741]
Hi, Is there any plan to keep this updated in future. I guess this is not maintained since last 2-3 years.
If yes, , is there any plan to upgrade this to MQTT version 4 in future please.

**Comment by trysten** [comment]
Yep. You can contribute via the milestone: https://github.com/homieiot/homie-esp8266/milestone/12
